Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM registry.access.redhat.com/rhscl/ruby-23-rhel7
+
+# sshd
+USER root
+RUN yum install -y --setopt=tsflags=nodocs openssh-server libicu-devel && \
+    yum clean all && /usr/sbin/sshd-keygen
+
+RUN mkdir /var/run/sshd
+#RUN sed -i 's/LogLevel INFO/LogLevel VERBOSE/' /etc/ssh/sshd_config
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
```

Comment:
Unfortunately I don't recall the reason for this change. My guess is ssh connections didn't work whatever way we do them from millicore.
It might be good to remove this line at some point and see what breaks.

The docs around this config haven't helped to jog my memory http://linux.die.net/man/8/pam_loginuid